content/docs/get-started/azure/deploy-changes.md

```diff
@@ -46,11 +46,11 @@ Do you want to perform this update? yes
 Updating (dev):
 
      Type                      Name                     Status      Info
-     pulumi:pulumi:Stack       update-az-templates-dev
- ~   └─ azure:storage:Account  storage                  updated     [diff: ~enableHttpsTrafficOnly]
+     pulumi:pulumi:Stack       quickstart-dev
+ ~   └─ azure:storage:Account  storage                  updated     [diff: ~tags]
 
 Outputs:
-    connectionString: "DefaultEndpointsProtocol=https;AccountName=storagefeda4143;AccountKey=...;EndpointSuffix=core.windows.net"
+    ~ connectionString: "DefaultEndpointsProtocol=https;AccountName=storagefeda4143;AccountKey=...;EndpointSuffix=core.windows.net" => output<string>
```

Comment:
This looks wrong... You are showing the result of an update here, after confirming with `yes`, so all outputs must have resolved by now. Did you mean to update the preview snippet?

Reply:
Whoops yeah, my bad (I suggested this change). This should've been in the preview snippet.

Reply:
Ah, sorry -- let me move that.